src/connectors/admin.js

```diff
@@ -12,6 +12,10 @@ export default function (fetch, apiUrl) {
     return { Authorization: `Bearer ${localStorage.getItem('accessToken')}` }
   }
 
+  const generateDeleteHeaders = () => {
+    return { Authorization: `Bearer ${localStorage.getItem('permission') || localStorage.getItem('accessToken')}` }
```

Comment:
Let's put it on "delete-permission-token" or something like that, so we can immediately know what we are looking at.